.github/workflows/_test.yaml

```diff
@@ -103,18 +103,12 @@ jobs:
           ALL_RUNNERS: |-
             {
               "trusted_builds": {
-                "amd64": [
-                  "ubuntu-latest"
-                ],
-                "arm64": [
-                  "ubuntu-latest-arm64-kong"
-                ]
+                "amd64": "ubuntu-latest-kong",
+                "arm64": "ubuntu-latest-arm64-kong"
               },
               "untrusted_builds": {
-                "amd64": [
-                  "ubuntu-latest"
-                ],
-                "arm64": []
+                "amd64": "ubuntu-latest",
+                "arm64": ""
```

Comment:
Wasn't there a way to do this with much simpler syntax? No yq just github expressions?

Reply:
Let me think about it

Reply:
https://github.com/kumahq/kuma/pull/10945#discussion_r1687751862

Reply:
For runners, I'll use GHA expressions.